sdk/communication/azure-communication-chat/tests/_shared/test_communication_identity_serializer.py

```diff
@@ -25,18 +26,33 @@ def test_missing_property_desrializer_throws(self):
                 kind=CommunicationIdentifierKind.CommunicationUser
```

Comment:
Typo in `test_missing_property_desrializer_throws(self)`

should be `test_missing_property_deserializer_throws(self)`